pkg/op-brains/op_brains/documents/optimism.py

```diff
@@ -82,10 +82,14 @@ def get_db_name(self) -> str:
 
 class ForumPostsProcessingStrategy(DocumentProcessingStrategy):
     @staticmethod
-    def retrieve():
-        out_db = connect_db.retrieve_data(
-            f'select "rawData", url, type, "externalId" from "{RAW_FORUM_DB}"'
-        )
+    def retrieve(only_not_summarized: bool = False):
+        if only_not_summarized:
+            query = f'select "rawData", url, type, "externalId" from "{RAW_FORUM_DB}" where "needsSummarize"=True'
+        else:
+            query = f'select "rawData", url, type, "externalId" from "{RAW_FORUM_DB}"'
+
+        out_db = connect_db.retrieve_data(query)
```

Comment:
https://linear.app/bleu-llc/issue/OP-99/refactor-op-brains-to-use-models-from-op-data-db

Reply:
think we could leave this as is if it's working 🤷